docs/releases.md

```diff
@@ -85,9 +121,14 @@ _Note:_ Since multiple software titles were listening on the ports `5000` and `8
 
         - When disabling the RPi camera feature, the bcm2835_isp kernel module is now additionally blacklisted. Since kernel 5.X it is otherwise loaded automatically and pulls in the whole camera modules stack as dependency, adding some additional memory usage and boot overhead. Many thanks to @ferbar for making us aware of this: <https://github.com/MichaIng/DietPi/issues/4203>
 
+### Changes / Improvements / Optimisations in 7.1
+
+- **`DietPi-Arr_to_RAM`** :octicons-arrow-right-16: Support for [Sonarr](../software/bittorrent/#sonarr) v3 and [Radarr](../software/bittorrent/#radarr) v3 has been added. On first link to RAM, a script `/mnt/dietpi_userdata/(sonarr|radarr|lidarr)/dietpi-arr_to_RAM.sh` is created, which allows updating the linked database backups via the programs "Custom Script" feature. Since v3 it is no longer possible to pass arguments to user-defined scripts or to call scripts within the `boot` directory, which broke the previous `/boot/dietpi/misc/dietpi-arr_to_RAM 2 (sonarr|radarr|lidarr)` calls in two ways.
```

Comment:
we would need to remove the ` around DietPi-Arr_to_RAM

Reply:
Agreed. I'll create a docs section for it by times, then we can make it a link. It requires adding "Arr" to the wordlist, to we can start using pirate speak, arrrr 😄.